requirements.txt

```diff
@@ -1,2 +1,2 @@
 https://github.com/wazo-platform/xivo-manage-db/archive/master.zip
-dbus==1.2.8
+dbus-python==1.2.16
```

Comment:
it seems to cause issue

Reply:
Yeah, it only causes issues now, because the other was never tested. There is no package `dbus`.

Reply:
It installs fine locally. It must be missing something on the CI... Looks like it requires dbus to build.

Reply:
Should I remove the test for installation (it was not there before) or install the dependency in the docker? A shame it won't resolve on its own.